endOfEpoch/shardchain/trigger.go

```diff
@@ -0,0 +1,41 @@
+package shardchain
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/endOfEpoch"
+)
+
+type ArgsNewShardEndOfEpochTrigger struct {
```

Comment:
comments on exported structs/methods
please check other places in this file

Reply:
done.